FhirToDataLake/SynapseValidate/Synapse_Validator.py

```diff
@@ -0,0 +1,207 @@
+import os
+import sys
+import argparse
+import concurrent.futures
+import json
+import requests
+import pyodbc
+import pandas as pd
+import time
+from urllib import parse as urlparse
+from datetime import datetime
+
+parser = argparse.ArgumentParser()
+parser.add_argument("--SynapseWorkspaceName", help="Name of Synapse workspace.")
+parser.add_argument("--FhirServerUrl", help="Fhir server url.")
+parser.add_argument("--Database", help="Name of database.")
+parser.add_argument("--SchemaCollectionDirectory", help="Schema collection directory path.")
+args = parser.parse_args()
+
+Sql_Server_Endpoint = args.SynapseWorkspaceName + "-ondemand.sql.azuresynapse.net"
+Database = args.Database
+Schema_Collection_Directory = args.SchemaCollectionDirectory
+Fhir_Server_Base_Url = args.FhirServerUrl
+Sql_Username = os.getenv('SQL_USERNAME')
+Sql_Password = os.getenv('SQL_PASSWORD')
+
+pyodbc.pooling = False
+
+class RawDataSource:
```

Comment:
Consider a better name, like `FhirDataClient`?

Reply:
Sure, renamed to FhirApiDataClient